### PR TITLE
BUG: Update WinProbe internal start and stop recording

### PR DESCRIPTION
**Pull request by RMR54**
This also fixes SetFocalPointDepth() tracebacks observed after freezing/unfreezing probe.

### CODE REVIEW

File: src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx [hunk 1]
```diff
@@ -692,7 +692,12 @@ PlusStatus vtkPlusWinProbeVideoSource::InternalConnect()
   std::this_thread::sleep_for(std::chrono::milliseconds(100));
   if(m_Mode == Mode::M)
   {
-    this->SetMModeEnabled(true);
+    SetMIsEnabled(true);
```
Comment (dzenanz) [inline src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx]: Calling `this->SetMModeEnabled(true)` would avoid code duplication. Please amend this commit.

Comment (RMR54) [reply src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx]: Do I remove the check `if(Connected)` within `setMModeEnabled`? Or, if I moved `this->SetMModeEnabled(true)`; to `InternalStartRecording()` is it okay to apply all M-Mode settings when unfreezing probe?

Comment (dzenanz) [reply src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx]: I forgot that during `InternalConnect`, connected is still `false`. Leave the code as-is.

**Comment by jamesobutler** [comment]
@adamrankin or @lassoan Could you merge this into master for us? Thanks!

**Pull request merged**